tools/accuracy_checker/accuracy_checker/annotation_converters/ms_coco.py

```diff
@@ -276,7 +334,17 @@ def _create_representations(
                 image, annotations,
                 label_id_to_label
             )
-            annotation = CoCoInstanceSegmentationAnnotation(image[1], segmentations, image_labels)
+
+            if not self.semantic_only:
+                annotation = CoCoInstanceSegmentationAnnotation(image[1], segmentations, image_labels)
+            else:
+                h, w, _ = image[2]
+                mask_file = self.masks_dir / "{:012}.png".format(image[0])
+                make_segmentation_mask(h, w, mask_file, image_labels, segmentations)
+                annotation = SegmentationAnnotation(image[1],
+                                                    str(mask_file.relative_to(self.masks_dir.parent)),
+                                                    mask_loader=GTMaskLoader.SCIPY)
```

Comment:
scipy uses pillow inside and used for byte manipulation in RGB image in VOC, for grayscale pillow will be enough
```suggestion
                                                    mask_loader=GTMaskLoader.PILLOW)
```